.docker/README.md

```diff
@@ -0,0 +1,78 @@
+# HarbourBridge: Docker Mode
+
+HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation and
+migration. This README provides details of HarbourBridge in docker mode. For general HarbourBridge information see
+this [README](https://github.com/cloudspannerecosystem/harbourbridge).
+
+### Before you begin
+
+1. [Install](https://cloud.google.com/sdk/docs/install) and [authorize](https://cloud.google.com/sdk/docs/authorizing#:~:text=If%20you%20want,grant%20access%20permissions.) the GCloud CLI using your Google Cloud account.
+2. Complete the steps in the [before you begin](https://github.com/cloudspannerecosystem/harbourbridge#before-you-begin) section in the HarbourBridge [README](https://github.com/cloudspannerecosystem/harbourbridge).
```

Comment:
This might be confusing to the user, this section specifies further links to `set up` (installing gcloud CLI etc.) and installing Go. Instead of hyperlinking, can we just add here whatever the user needs to do?
It will also improve reader flow, not having to jump to different URLs.

Reply:
True, sure.

Reply:
Done. Haven't elaborated the GCloud CLI and Docker installations because they would increase the length of README and shift focus from the core prerequisite steps for running HarbourBridge in docker.